CMakeLists.txt

```diff
@@ -166,7 +167,8 @@ if(SIMPLNX_DOWNLOAD_TEST_FILES)
   add_custom_target(Fetch_Remote_Data_Files ALL
     COMMAND "${CMAKE_COMMAND}" -DCMAKE_CONFIG=$<CONFIG> -P "${FETCH_FILE_PATH}"
     COMMENT "Downloading Test Data Files"
-  #  WORKING_DIRECTORY "${TEST_WORKING_DIR}"
```

Comment:
There are a lot of unrelated formatting changes in this file. I think we should reapply on the actual changes for the PR.

Reply:
I tried to revert most of the formatting changes...